src/configs/cloudinary.js

```diff
@@ -0,0 +1,9 @@
+const cloudinary_env = process.env.CLOUDINARY_ENV;
+const cloudinary_api_key = process.env.CLOUDINARY_API_KEY;
+const cloudinary_api_secret = process.env.CLOUDINARY_API_SECRET;
+
+module.exports = {
+  cloudinary_env,
+  cloudinary_api_key,
+  cloudinary_api_secret,
+};
```

Comment:
Can we keep these variables camel case, also change wherever they are used in application

Reply:
done